gclient/src/api/calls.rs

```diff
@@ -533,15 +533,18 @@ impl GearApi {
         &self,
         args: impl IntoIterator<Item = MessageId> + Clone,
     ) -> Result<(Vec<Result<u128>>, H256)> {
-        let mut values = BTreeMap::new();
-        for message_id in args.clone().into_iter() {
-            values.insert(
-                message_id,
-                self.get_mailbox_message(message_id)
-                    .await?
-                    .map(|(message, _interval)| message.value()),
-            );
-        }
+        let message_ids: Vec<MessageId> = args.clone().into_iter().collect();
```

Comment:
for all changes auto-guessing types still valid 
```suggestion
        let message_ids: Vec<_> = args.clone().into_iter().collect();
//      or
//      let message_ids = args.clone().into_iter().collect::<Vec<_>>();
```